remme/shared/forms/_validators.py

```diff
@@ -49,6 +49,32 @@ def __call__(self, form, field):
             raise validators.StopValidation(message)
 
 
+class BooleanTypeRequired(object):
+    """
+    Validates the value for the required integer type.
```

Comment:
Update documentation according to the logic.

Reply:
Done.